ingest/src/test/java/com/microsoft/azure/kusto/ingest/IngestClientImplTest.java

```diff
@@ -339,4 +340,16 @@ private String getSampleResultSetDump() {
                 "\"1\",\"leo\"\n\"2\",\"yui\"\n" :
                 "\"1\",\"leo\"\r\n\"2\",\"yui\"\r\n";
     }
+
+    @Test
+    void IngestFromBlob_IngestionReportMethodIsTable_RemovesSecrets() throws Exception {
```

Comment:
I would move this test to a new file dedicated to the new class you've created. it then can be scaled down to testing that everything is stripped like you do with the url comparison. No need to do that test here in the main class (IMO)